rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -144,10 +131,8 @@ void opcode_0009(const ScriptArguments& args, ScriptFloatGlobal arg1G, const Scr
 	@arg arg2 
 */
 void opcode_000a(const ScriptArguments& args, ScriptIntLocal arg1L, const ScriptInt arg2) {
-	RW_UNIMPLEMENTED_OPCODE(0x000a);
-	RW_UNUSED(arg1L);
-	RW_UNUSED(arg2);
 	RW_UNUSED(args);
+	arg1L += arg2;
```

Comment:
What does the `%2h%` stand for? Does that somehow imply this only works with halfwords?
Can we assert parameter types?